lib/tarantool_server.py

```diff
@@ -1016,16 +1016,6 @@ def stop(self, silent=True, signal=signal.SIGTERM):
             color_log('Server [%s] start against running ...\n',
                       schema='test_var')
             return
```

Comment:
Could you cover a case by test?

Reply:
If you mean a case covering the issue with hanging, I am going to do it in the next commit.

Reply:
Test is added.

Reply:
Are you added it in a separate commit intentionally? Sometimes we add a fix and a test for it in a single commit.

Reply:
Yes. Actually, I don't see any issue that the test is in a separate commit. I have two commits (Add timeout for starting tarantool server and Kill processes of non-started tarantool servers) that fix two issues. And I have the commit with the test (Add unit test checking no hanging procs present) covering mentioned commits. And IMHO it is OK.